.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -83,7 +83,9 @@ Check that the release will work:
 - [ ] Update crate versions, commit the changes to the release branch, and do a release dry-run:
 
 ```sh
-cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad beta
+cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad --exclude zebra-scan beta
+fastmod -d zebra-scan -e toml -F '0.1.0-alpha.1' '0.1.0-alpha.2' # Update this command if zebra-scan is still in alpha and alpha version bumps won't work with cargo-release
+# cargo release version --verbose --execute --allow-branch '*' --package zebra-scan beta # [ major | minor | patch ]
```

Comment:
That's strange, `alpha` is a documented level in `cargo release`:
https://github.com/crate-ci/cargo-release/blob/master/docs/reference.md#cli-arguments

It might be worth reporting this as a bug at https://github.com/crate-ci/cargo-release/issues/new

For now I'd suggest using an explicit version, which I've tested and it works:
```suggestion
# Due to a bug in cargo-release, we need to pass an exact version here
cargo release version --verbose --execute --allow-branch '*' --package zebra-scan 0.1.0-alpha.1
```


```
$ cargo release version --execute --allow-branch '*' --package zebra-scan 0.1.0-alpha.1
warning: uncommitted changes detected, please resolve before release:
  cache_dir/ (WT_NEW)
  zebra-network/proptest-regressions/peer_set/set/ (WT_NEW)
  zebrad.toml (WT_NEW)
Bump zebra-scan 0.1.0-alpha.1? [y/N] 
y
   Upgrading zebra-scan from 0.1.0-alpha.0 to 0.1.0-alpha.1
    Updating zebrad's dependency from 0.1.0-alpha.0 to 0.1.0-alpha.1
```